README.md

```diff
@@ -109,15 +109,15 @@ People
 * [Xiaohang Lu](people/xiaohang-lu.md) - xl672 - [https://s3.us-east-2.amazonaws.com/morning-shawn.com/grading.html](https://s3.us-east-2.amazonaws.com/morning-shawn.com/grading.html)[![CircleCI](https://circleci.com/gh/LuXiaohang/skeleton.svg?style=svg)](https://circleci.com/gh/LuXiaohang/skeleton)
-* [Yingxiang Lu](people/yingxiang-lu.md) - yl2749 - [http://aaronlyx.com.s3-website.us-east-2.amazonaws.com/grading.html](http://aaronlyx.com.s3-website.us-east-2.amazonaws.com/grading.html) 
+* [Yingxiang Lu](people/yingxiang-lu.md) - yl2749 - [http://aaronlyx.com.s3-website.us-east-2.amazonaws.com/grading.html](http://aaronlyx.com.s3-website.us-east-2.amazonaws.com/grading.html)
 * [Lixuan Mao](people/lixuan-mao.md) - lm769 - [http://ec2-18-221-74-142.us-east-2.compute.amazonaws.com:8080](http://ec2-18-221-74-142.us-east-2.compute.amazonaws.com:8080)[![CircleCI](https://circleci.com/gh/thumlx12/skeleton.svg?style=svg)](https://circleci.com/gh/thumlx12/skeleton)
 * [Yunie Mao](people/yunie-mao.md) - ym224 - [http://ec2-34-205-90-110.compute-1.amazonaws.com](http://ec2-34-205-90-110.compute-1.amazonaws.com) [![CircleCI](https://circleci.com/gh/ym224/CS5356_Assignment2.svg?style=svg)](https://circleci.com/gh/ym224/CS5356_Assignment2)
 * [Jake Magid](people/jake-magid.md) - jm2644 - [http://cs5356jm.us.s3-website-us-east-1.amazonaws.com/grading.html](http://cs5356jm.us.s3-website-us-east-1.amazonaws.com/grading.html) [![CircleCI](https://circleci.com/gh/jmagid15/skeleton.svg?style=svg)](https://circleci.com/gh/jmagid15/skeleton)
 * [Fani Maksakuli](people/fani-maksakuli.md) - fm399 - [http://fanisbucket.s3-website.us-east-2.amazonaws.com/grading.html](http://fanisbucket.s3-website.us-east-2.amazonaws.com/grading.html)
 * [Thomas Matecki](people/thomas-matecki.md) - tpm93 - [https://s3.amazonaws.com/cs5356-tpm93/grading.html](https://s3.amazonaws.com/cs5356-tpm93/grading.html)
 * [Yuxue Mei](people/yuxue-mei.md) - ym434 - [http://ec2-yuxue2017.com.s3-website.us-east-2.amazonaws.com/grading.html](http://ec2-yuxue2017.com.s3-website.us-east-2.amazonaws.com/grading.html)[![CircleCI](https://circleci.com/gh/Detective2016/skeleton.svg?style=svg)](https://circleci.com/gh/Detective2016/skeleton)
-* [Andi Mitre](people/andi-mitre.md) - am2886 - [http://startup-sys-andi.s3-website-us-east-1.amazonaws.com/grading.html](http://startup-sys-andi.s3-website-us-east-1.amazonaws.com/grading.html)
-* [Yian Mo](people/yian-mo.md) - ym443 - [http://yian.ml.s3-website.us-east-2.amazonaws.com/grading.html](http://yian.ml.s3-website.us-east-2.amazonaws.com/grading.html)[![CircleCI](https://circleci.com/gh/moyk/skeleton.svg?style=svg)](https://circleci.com/gh/moyk/skeleton) 
+* [Andi Mitre](people/andi-mitre.md) - am2886 - [http://ec2-54-234-72-129.compute-1.amazonaws.com](http://ec2-54-234-72-129.compute-1.amazonaws.com) [![CircleCI](https://circleci.com/gh/andimitre/skeleton.svg?style=svg)](https://circleci.com/gh/andimitre/skeleton)
```

Comment:
Whoops!

Reply:
@amfleming I only see a space difference on the one below line 120?

Reply:
@amfleming updated this PR without spacing issues here 
https://github.com/CT-CS5356-Fall2017/cs5356/pull/388